{"error": null, "issues": [{"address": 86, "code": "  uint to", "debug": "", "description": "Function fallback retrieves the transaction origin (tx.origin) using the ORIGIN opcode. Use msg.sender instead.\nSee also: https://solidity.readthedocs.io/en/develop/security-considerations.html#tx-origin", "filename": "/temp/SoPoW-0.sol", "function": "fallback", "lineno": 76, "title": "Use of tx.origin", "type": "Warning"}, {"address": 1228, "code": "_payment = getPaymen", "debug": "storage_2: 0x1d89\nblock_number: 0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffdc7c\nretval_1131: 0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff\norigin: 0x0\ncaller: 0x0\ncalldatasize_Sopow: 0x3\n", "description": "A possible integer overflow exists in the function `fallback`.\nThe addition or multiplication may result in a value higher than the maximum representable integer.", "filename": "/temp/SoPoW-0.sol", "function": "fallback", "lineno": 116, "title": "Integer Overflow ", "type": "Warning"}, {"address": 2097, "code": " _target, uint _total, ", "debug": "storage_2: 0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffdae0\nblock_number: 0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffdae0\norigin: 0x0\ncaller: 0x0\ncalldatasize_Sopow: 0x3\n", "description": "A possible integer overflow exists in the function `fallback`.\nThe addition or multiplication may result in a value higher than the maximum representable integer.", "filename": "/temp/SoPoW-0.sol", "function": "fallback", "lineno": 58, "title": "Integer Overflow ", "type": "Warning"}], "success": true}
